lib/route/neigh.c

```diff
@@ -1066,6 +1078,19 @@ int rtnl_neigh_get_master(struct rtnl_neigh *neigh) {
 	return neigh->n_master;
 }
 
+void rtnl_neigh_set_nhid(struct rtnl_neigh *neigh, unsigned int nhid)
+{
+	neigh->n_nhid = nhid;
+	neigh->ce_mask |= NEIGH_ATTR_NHID;
+}
+
+int rtnl_neigh_get_nhid(struct rtnl_neigh *neigh) {
+	if (neigh->ce_mask & NEIGH_ATTR_NHID)
+		return neigh->n_nhid;
+	else
+		return -1;
```

Comment:
`return -NLE_NOATTR;`

Reply:
can we do the error-paths first and return-early (not if/else)?
```c
    if (!(neigh->ce_mask & NEIGH_ATTR_NHID))
        return -NLE_NOATTR;
    return neigh->n_nhid;
```

Reply:
done